components/Main/Pages/TimerPage.js

```diff
@@ -1,30 +1,55 @@
-//@todo Michael please integrate the timer component there.
 import React from 'react';
-import {View, Text, StyleSheet, Button} from 'react-native';
+import {Text, View, StyleSheet, TouchableOpacity, Dimensions,Image} from 'react-native';
+import LinearGradient from 'react-native-linear-gradient';
 
 const TimerPage = ({navigation}) => {
-  return(
-    <View style={styles.header}>
-      <Text style={styles.text}>Timer starts!</Text>
-      <Button title = "End Timer" color="red" 
-           onPress={() => navigation.navigate('Status Report')} />
+  return (
+    <View style={styles.container}>
+      <LinearGradient
+          colors={['#C8E0E1', 'white']}
+          style={styles.linearGradient}
+        >
+        <Text style={styles.time}>01:26</Text>
+        <TouchableOpacity style = {styles.button} onPress={() => navigation.navigate('Status Report')}>
```

Comment:
Disproportional Button. Can you create a function that adjust the size of the button base on the screen size used by the phone? Given that we are using images we need to ensure the button size is proportional to the screen size.